complete/build.gradle

```diff
@@ -53,11 +53,13 @@ dependencies {
     compile "org.grails.plugins:gsp"
     console "org.grails:grails-console"
     profile "org.grails.profiles:web"
+    runtime "org.glassfish.web:el-impl:2.1.2-b03"
```

Comment:
its one of the default dependencies of grails 3.3.0.

https://github.com/erichelgeson/grails-versions/compare/3.2.8...3.3.0